app/models/concerns/media_scraping_bot_item.rb

```diff
@@ -0,0 +1,108 @@
+module MediaScrapingBotItem
+  extend ActiveSupport::Concern
+
+  class ScrapingBotError < StandardError; end
+  class ScrapingBotResponseError < StandardError; end
+
+  Media.class_eval do
+    # Step 1: Send POST request to ScrapingBot to start scraping
+    def self.scrapingbot_start_request(resource, url)
+      scrapingbot_url = "https://api.scraping-bot.io/scrape/data-scraper"
+
+      # Configure the POST request based on the resource (Facebook or Instagram)
+      scraper_name = case resource
+                     when :facebook_profile then "facebookProfile"
+                     when :facebook_post then "facebookPost"
+                     when :instagram_profile then "instagramProfile"
+                     when :instagram_post then "instagramPost"
+                     else
+                       raise ScrapingBotError, "Unsupported resource: #{resource}"
+                     end
```

Comment:
Just a suggestion to use array for this mapping
i.e.
resource_mapping =  {facebook_profile: 'facebookProfile',  facebook_post: 'facebookPost',  instagram_profile: 'instagramProfile', instagram_post: 'instagramPost'}
raise ScrapingBotError, "Unsupported resource: #{resource}" if resource_mapping[resource].blank?
scraper_name = resource_mapping[resource]